run_all_tests.sh

```diff
@@ -0,0 +1,2 @@
+
+sbt clean scalastyleAll compile coverage test it:test coverageOff coverageReport dependencyUpdates
```

Comment:
`scalastyleAll` is a command alias for running scalastyle in app, tests and it test files. You'll need to add this in the build.sbt file - see example here: [hmrc/maintain-beneficiaries-frontend](https://github.com/hmrc/maintain-beneficiaries-frontend/blob/2c572dae7267ff69f8875d931ad6ede22f6406c9/build.sbt#L72)

I think you can remove the it:test since this repo only has unit tests.